RELEASING.md

```diff
@@ -2,8 +2,14 @@
 
 We're following [semantic versioning](https://semver.org/) approach to releases in Kritis.
 
+## Create a release tracking issue
+
+Use the [release tracking](https://github.com/grafeas/kritis/issues/new?template=feature_request.md)
+template to create a release tracking issue. Use this issue to note any problems that come up, as well
```

Comment:
could you pls elaborate on this? are we expecting to document why the release is needed, due to problems?

Reply:
Done.